src/components/PresentBar/PresentBar.test.tsx

```diff
@@ -266,4 +273,88 @@ describe('<PresentBar/>', () => {
       }),
     ).toBeInTheDocument();
   });
+
+  it('should be able to end presentation of another user if can moderate', async () => {
```

Comment:
I kind of miss a test for the admin, but feel like it is arguable that if the moderator can, the admin will also be able.

Reply:
yea, agreed that it is fine (admin can moderate)